addons/website_livechat/controllers/main.py

```diff
@@ -66,3 +66,9 @@ def get_session(self, channel_id, anonymous_name, previous_operator_id=None, cha
         if visitor_sudo:
             anonymous_name = visitor_sudo.with_context(lang=visitor_sudo.lang_id.code).display_name
         return super(WebsiteLivechat, self).get_session(channel_id, anonymous_name, previous_operator_id=previous_operator_id, chatbot_script_id=chatbot_script_id, **kwargs)
+
+    def _livechat_templates_get(self, **kwargs):
+        return [
+            *super(WebsiteLivechat, self)._livechat_templates_get(**kwargs),
```

Comment:
no need for kwargs that we know are not necessary, also strange to use * to spread in python, I don't see it often in our code

Reply:
change done